File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/RevokeTrustedDeviceCommand.java
package com.happs.ximand.ringcontrol.model.object.command;

public class RevokeTrustedDeviceCommand extends BluetoothCommand {

    private static final byte REVOKE_TRUSTED_DEVICE_COMMAND_CODE = 2;

    protected RevokeTrustedDeviceCommand() {
        super(REVOKE_TRUSTED_DEVICE_COMMAND_CODE);
    }

    @Override
    public byte[] getCommand() {
        return null;
    }
}

File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/RevokeTrustedDeviceCommand.kt
package com.happs.ximand.ringcontrol.model.`object`.command

class RevokeTrustedDeviceCommand : BluetoothCommand<String?>(REVOKE_TRUSTED_DEVICE_COMMAND_CODE) {
    override fun toByteArray(): ByteArray {
        return ByteArray(0)
    }

    override fun getMainContent(): String {
        throw UnsupportedOperationException()
    }

    companion object {
        private const val REVOKE_TRUSTED_DEVICE_COMMAND_CODE: Byte = 2
    }
}